searx/static/themes/__common__/less/stats.less

```diff
@@ -1,60 +1,54 @@
-.issue-hide{
-    display: none;
-}
-
```

Comment:
The file is untabify.

Is it possible not to add tab ?
I guess it is more complicated than `(setq-default indent-tabs-mode nil)` ?

Reply:
Sorry, I'm sloppy and use the emacs defaults for new created files / For better tab handling I set `indent-tabs-mode` in c0c71ae.